assembly/guide-configs/app-dev/features.json

```
@@ -585,6 +585,64 @@
         }
       ]
     },
+    {
+      "type": "h2",
+      "id": "app-dev-context/side-effects"
+    },
+    {
+      "type": "langtabs",
+      "langtabs": [
+        {
+          "lang": "go",
+          "id": "go/how-to-execute-a-side-effect-in-go"
+        },
+        {
+          "lang": "java",
+          "id": "java/how-to-use-a-side-effect-in-java"
+        },
+        {
+          "lang": "php",
+          "id": "none"
+        },
+        {
+          "lang": "python",
+          "id": "none"
+        },
+        {
+          "lang": "typescript",
+          "id": "none"
```

```suggestion
          "id": "na"
```

and generator fills in with "Not applicable to this SDK." instead of "Content is currently unavailable."